modules/planning/tasks/optimizers/path_decider/path_decider.cc

```diff
@@ -173,15 +173,17 @@ bool PathDecider::MakeStaticObstacleDecision(
                                                obstacle->Id(), object_decision);
       }
     } else if (FLAGS_enable_nudge_decision) {
-      // nudge
-      if (curr_l > sl_boundary.end_l()) {
+      // 3. NUDGE if laterally very close.
+      if (sl_boundary.end_l() < curr_l - min_nudge_l) {  // &&
+          // sl_boundary.end_l() > curr_l - min_nudge_l - 0.3) {
```

Comment:
Refine the comments, tidy and serious.

Reply:
Tomorrow we will do road test. This is for the purpose of easily switching between different configurations. I will clean things up when road test completes.

Reply:
It's not a good practice though. A solution might be to switch with FLAGS, so that you don't need to rebuild.
And at least a TODO to clarify that it will be tested and cleaned up later.